lib/reply.js

```diff
@@ -195,6 +197,12 @@ Reply.prototype.headers = function (headers) {
   return this
 }
 
+Reply.prototype.disableStringify = function (flag) {
```

Comment:
```suggestion
Reply.prototype.skipSerialization = function () {
```

And simply set the trackers to `true` such that `reply.skipSerialization().send('{"foo":"bar"}')` works.

Reply:
Sounds good, i'll change this.